front/pages/api/w/[wId]/assistant/agent_configurations/index.ts

```diff
@@ -72,9 +72,13 @@ async function handler(
         });
       }
       // extract the view from the query parameters
-      const queryValidation = GetAgentConfigurationsQuerySchema.decode(
-        req.query
-      );
+      const queryValidation = GetAgentConfigurationsQuerySchema.decode({
```

Comment:
why not add the limit in GetAgentConfsQuerySchema?

Reply:
The `GetAgentConfigurationsQuerySchema` already specifies `limit`. However, NestJS, which is built on ExpressJs, doesn't automatically parse URL query parameters. Therefore, a parameter like `limit=10` is interpreted as a string. In this case, we manually parse it as a number if it is provided.